### PR TITLE
Add player by :id endpoint

### PR DESCRIPTION
**Pull request by daelynj**
We can `GET` a specific player by `:id` by following the route ie. `/players/4`.
We also handle the error when a player does not exist.

### CODE REVIEW

File: apps/api/controllers/players/show.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+        expose :player
+
+        def initialize(interactor: Interactors::Players::FetchPlayer.new)
+          @interactor = interactor
+        end
+
+        def call(params)
+          player = @interactor.call(params[:id]).player
+          halt 400 && self.status = 400 if player.nil?
```
Comment (jgayfer) [inline apps/api/controllers/players/show.rb]: We probably want to use `404` here (not found).

Comment (jgayfer) [reply apps/api/controllers/players/show.rb]: Do we need to explicitly set `status` when using `halt`?

File: spec/api/controllers/players/show_spec.rb [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    before do
+      repository.clear
+      repository.create(id: '1')
+      repository.create(id: '2')
+    end
+
+    it 'is successful' do
+      response = action.call(id: 1)
+
+      expect(response[0]).to eq(200)
```
Comment (jgayfer) [inline spec/api/controllers/players/show_spec.rb]: What is `response[0]`? Hard to understand what we're testing here.

File: spec/typinggame_server/interactors/players/fetch_player_spec.rb [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  let(:params) { 1 }
+
+  before do
+    repository.clear
+
+    repository.create(id: '1')
+    repository.create(id: '2')
+  end
+
+  context 'good input' do
```
Comment (jgayfer) [inline spec/typinggame_server/interactors/players/fetch_player_spec.rb]: Let's use a more description context block.

ie. `When the player exists`

File: apps/api/controllers/players/show.rb (hunk 1 above)
Comment (daelynj) [reply apps/api/controllers/players/show.rb]: @jgayfer fixed.

File: spec/api/controllers/players/show_spec.rb (hunk 2 above)
Comment (daelynj) [reply spec/api/controllers/players/show_spec.rb]: @jgayfer fixed.

File: spec/typinggame_server/interactors/players/fetch_player_spec.rb (hunk 3 above)
Comment (daelynj) [reply spec/typinggame_server/interactors/players/fetch_player_spec.rb]: @jgayfer fixed.

**Pull request merged**